shop/forms.py

```diff
@@ -0,0 +1,84 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from .models import Product, User, ReturnRequest, Purchase
+
+
+class ProductForm(forms.ModelForm):
+    class Meta:
+        model = Product
+        fields = ['name', 'description', 'price', 'image', 'stock']
```

Comment:
It's a bad practice to use mixed single and double quotes within one project or module.